src/components/app/data/services/bffs.ts

```diff
@@ -86,7 +95,9 @@ export async function makeBFFRequest({
 
     // Make request to BFF.
     const result = await getAuthenticatedHttpClient().post(url, params);
+    const responseData = _cloneDeep(result.data);
```

Comment:
I'm not sure we necessarily need to `_cloneDeep(result.data)` here vs. simply doing something like:

```tsx
const  {
  catalog_uuids_to_catalog_query_uuids,
  ...originalResponseData
} = result.data;
const response = {
  ...camelCaseObject(originalResponseData),
  catalogUuidsToCatalogQueryUuids: catalog_uuids_to_catalog_query_uuids,
};
```

One added benefit of this example is that the `catalog_uuids_to_catalog_query_uuids` is effectively removed from `camelCaseObject(originalResponseData)`.

We're not mutating this original `responseData`, so not quite what value the `_cloneDeep` is providing?